roles/collectd/tasks/main.yml

```diff
@@ -1,12 +1,17 @@
 ---
 - name: install collectd
-  yum:
-      name: '{{ item }}'
-      state: present
-  with_items:
-      - collectd
```

Comment:
Well, that was a question actually, not a stand :), but thanks! I would also suggest to use our variable from defaults, so s/collectd/{{ collectd_package_name }}/